alacritty/src/renderer/text/builtin_font.rs

```diff
@@ -1,5 +1,6 @@
-//! Hand-rolled drawing of unicode [box drawing](http://www.unicode.org/charts/PDF/U2500.pdf)
-//! and [block elements](https://www.unicode.org/charts/PDF/U2580.pdf), and also powerline symbols.
+//! Hand-rolled drawing of unicode [box drawing](http://www.unicode.org/charts/PDF/U2500.pdf),
+//! [block elements](https://www.unicode.org/charts/PDF/U2580.pdf)
+//! and [sextants](https://www.unicode.org/charts/PDF/U1FB00.pdf), and also powerline symbols.
```

Comment:
This comment should be reworded to not include what exactly is being drawn. Otherwise we'll have to change it constantly.